index.html

```diff
@@ -0,0 +1,344 @@
+            <li>
+                <button class="text">
+                    html
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    css
+                </button>
+            </li>
+            <li>
+                <button class="text">
+                    javaScript
+                </button>
+            </li>
+        </ul>
+        <div class="see-more">
+
+            <button class="text-1">
+                See Project
+            </button>
+        </div>
+
+
+        </div>
+
+    </section>
+    <section class="About-myself">
+        <h1 class="intro">
+            About Myself
+        </h1>
+        <p class="about-1">
+            Hello I’m a software developer! I can <br> help you build a product , feature or <br> website Look through some of my work <br>and experience! If you like what you see <br>and have a project you need coded, <br> don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+            <h2>LET'S CONNECT</h2>
+            <div class="icons">
+                <ul class="social-media">
+                    <li>
+                        <a href=""> <i class="fa-brands fa-twitter fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands  fa-linkedin fa-1x "></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-github fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""> <i class="fa-brands fa-m fa-1x"></i></a>
+                    </li>
+                    <li>
+                        <a href=""><i class="fa-solid fa-hand-peace fa-1x"></i></a>
+                    </li>
+                </ul>
+
+            </div>
+            <div class="get-more">
+
+                <button class="text-2">
+                    Get my resume
+                </button>
+            </div>
+        </div>
+        </div>
+        <div class="p-skills">
+            <h1 class="languages">
+                Languages
+                <i class='fas'>&#xf078;</i>
+            </h1>
+
+            <ul class="skills-1">
+                <li>
+                    <img src="images/Frame 9.png" alt="">
+        
+                <li>
+                    <img src="images/Frame 10.png" alt="">
+                </li>
+                <li>
+                    <img src="images/Frame 11.png" alt="">
+                    
+                </li>
+            </ul>
+            <div class="other-skills">
+                <h1 class="frame-works">
+                    Frameworks
+                    <i class='fas'>&#xf054;</i>
+                </h1>
+                <hr>
+                <h1 class="skills-2">
+                    Skills
+                    <i class='fas'>&#xf054;</i>
+                </h1>
+                <hr>
+            </div>
```

Comment:
- Kindly it should be better to add some padding to this section title (Langues, Frameworks, Skills) to make it look like the Figma design.
![image](https://user-images.githubusercontent.com/95297251/206455262-5d5134a6-5c11-4410-81eb-87a7bc423db0.png)